cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -225,6 +225,138 @@ def test_schedulable_memory_validator(schedulable_memory, ec2memory, instance_ty
             "Instance types listed under Compute Resource TestComputeResource must have the same number of CPU "
             "cores when Simultaneous Multithreading is disabled.",
         ),
+        # Instance Types should have the same number of GPUs
```

Comment:
Provide all the tests where there is no failure, e.g. same count, same manufacturer is respected